app/frontend/src/components/Icon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import CakeIcon from "@material-ui/icons/CakeOutlined";
+import EmojiPeopleIcon from "@material-ui/icons/EmojiPeopleOutlined";
+import TranslateIcon from "@material-ui/icons/TranslateOutlined";
+import LocationIcon from "@material-ui/icons/LocationOnOutlined";
+import WorkIcon from "@material-ui/icons/WorkOffOutlined";
+
+interface IconProps {
+  type: "cake" | "gender" | "location" | "language" | "work";
+  titleAccess: string;
+}
+
+export default function Icon({ type, titleAccess }: IconProps) {
+  switch (type) {
+    case "cake":
+      return <CakeIcon titleAccess={titleAccess} />;
+    case "gender":
+      return <EmojiPeopleIcon titleAccess={titleAccess} />;
+    case "location":
+      return <LocationIcon titleAccess={titleAccess} />;
+    case "language":
+      return <TranslateIcon titleAccess={titleAccess} />;
+    case "work":
+      return <WorkIcon titleAccess={titleAccess} />;
```

Comment:
Any ideas about a better way to do this that doesn't embed the material ui icons everywhere in our code? (I imagine we won't use them ultimately)

Reply:
I think making an icons folder within an assets folder with an index.ts that spits all icons out.

Reply:
Not sure I completely understand the question - do you mean not having to write out the specific Icon components (like `EmojiPeopleIcon`) as the list of icon options grow within this component?

Reply:
@darrenvong yes. I like @amdwit's suggestion